output/createOPeNDAPFileList.sh

```diff
@@ -85,7 +85,7 @@ for file in $fcstFile $bestFile ; do
       cp $file . 2>&1 > errmsg || warn "cycle $CYCLE: $SCENARIO: $THIS: Could not link to $file: `cat $errmsg`"
    fi
 done
-ceraNonPriorityFiles=( `ls $CONFIG $SYSLOG $CYCLELOG $SCENARIOLOG cpra.post.log *.csv endrisinginun.63.nc everdried.63.nc fort.64.nc fort.68.nc fort.71.nc fort.72.nc fort.73.nc initiallydry.63.nc inundationtime.63.nc maxinundepth.63.nc maxrs.63.nc maxvel.63.nc minpr.63.nc rads.64.nc swan_DIR.63.nc swan_DIR_max.63.nc swan_TMM10.63.nc swan_TMM10_max.63.nc 2>> $SCENARIOLOG` )
```

Comment:
isn't this why we added `postExcludeFiles`?

Reply:
any chance we can rename the variables to be more generic rather than who they were added for? Otherwise, the change is fine by me!